### DISCUSSION TITLE
Non multiome data with only matching sample labels

### DISCUSSION THREAD

**Discussion by atlas1584** [aertslab/scenicplus#215] [Q&A]
Hello,

I have a scRNA seq for a set of cells collected at different time points and with 2 treatment conditions (ie DrugA_time0, DrugB_time0, DrugA_time1, DrugB_time1,  DrugA_time2, DrugB_time2). I have scATACseq data from the same timepoints and treatment conditions (with same labels in the parantheses above) but the data is not exactly from the same cells, so it will be a pseudo-multiome data. 

My challenge is that while running scenic+, I only have the "labels" that match between the two datasets ( ie DrugA_time0) to generate meta cells. Also, these cells are all from one specific cell type so I am unable to assign cell states which means I can't use cell type to create metacells. In the melanoma tutorial, you have "MM lines; each cell line comes from a different patient" and you mention: "The key thing to be able to do this is that the scATAC-seq cells are annotated to cell types/states and that these labels are shared with the scRNA-seq annotation. ". so in my case there won't be multiple different cell types or cells coming from different patients.

My question is whether it is still a good idea to run scenicplus on this dataset or whether I should just stick to pySCENIC. Ideally I would want to find differential eRegulons between drugs at each time point (DrugA_time0 vs DrugB_time0) and also see the overall differences between DrugA and DrugB. I am concerned that I will be generating the metacells from the same variable that I will then use for differential comparisons. I would appreciate if you could offer any advice about whether you have tested the tool for a situation like mine and if scenic+ would be powered to perform this analysis in a dataset like this.

Thanks

**Comment by SeppeDeWinter** [comment]
Hi @atlas1584 

I have not performed the specific type of analysis you are mentioning, so I can not comment on how well SCENIC+ will perform in this particular case. 

I would give it a shot though. As you mentioned, you can match the scATAC-seq and scRNA-seq cells based on the type of treatment and time point. I would indeed use that label to match the cells across modalities.

Good luck with the analysis. Hopefully you will get useful results.


Best,

Seppe
